.github/workflows/build-fb-image.yaml

```diff
@@ -36,18 +45,100 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
 
       - name: Login to Docker Hub
         uses: docker/login-action@v3
         with:
+          registry: docker.io
```

Comment:
Are we pushing to docker.io now instead? I would keep things in ghcr.io if you can particularly for PRs otherwise we have to start dealing with rate limiting.